src/ray/util/url.cc

```diff
@@ -0,0 +1,66 @@
+#include <stdlib.h>
+
+#include <algorithm>
+#include <string>
+
+#include <boost/asio/ip/address.hpp>
+#include <boost/asio/ip/tcp.hpp>
+
+#include "ray/util/logging.h"
+#include "ray/util/url.h"
+
+boost::asio::ip::tcp::endpoint parse_ip_tcp_endpoint(
```

Comment:
This function should be documented. Not sure what it does.

Reply:
It just [parses the host & port of a URL](/ray-project/ray/pull/6823/files/d8ad4c4a1569c264c82226eb19a0794298089bc1#diff-a67b85512722e126b455ba21a0c99349R15). I added a comment to the header.